CalibTracker/SiPixelQuality/plugins/SiPixelStatusProducer.cc

```diff
@@ -0,0 +1,357 @@
+/**_________________________________________________________________
+   class:   SiPixelStatusProducer.cc
+   package: CalibTracker/SiPixelQuality/SiPixelStatusProducer
+
+   author:
+
+   ________________________________________________________________**/
+
+
+// C++ standard
+#include <string>
+// ROOT
+#include "TMath.h"
+
+// CMSSW FW
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/Framework/interface/ESWatcher.h"
+
+// CMSSW DataFormats
+#include "DataFormats/Common/interface/ConditionsInEdm.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/FEDRawData/interface/FEDNumbering.h"
+#include "DataFormats/SiPixelCluster/interface/SiPixelCluster.h"
+#include "DataFormats/SiPixelDetId/interface/PXBDetId.h"
+#include "DataFormats/SiPixelDetId/interface/PXFDetId.h"
```

Comment:
Hi @tocheng , could you please remove the includes of PXB and PXFDetId ? As far as I can see you are not using those classes and anyhow are not appropriate for pixel Phase1, thanks

Reply:
Hello @boudoul, many thanks for pointing it out. It is fixed in the latest commit.